packages/core/src/styles/tokens/tokens.stories.ts

```diff
@@ -63,31 +59,33 @@ function getTokenTable(key: string, globals: any = {}) {
         })}
       </tbody>
     </table>
-    <table
-      class="token-table"
-      cds-layout="display:none display@sm:block"
-      cds-table="border:row border:outside"
-      cds-text="left"
-    >
-      <thead>
-        <tr cds-layout="grid cols:4">
-          <th cds-layout="p-y:md p-x:lg">Token</th>
-          <th cds-layout="p-y:md p-x:lg">Value</th>
-          <th cds-layout="p-y:md p-x:lg">Demo</th>
-        </tr>
-      </thead>
-      <tbody>
-        ${getTokens(key, globals).map((token: any) => {
-          return html`<tr cds-layout="grid cols:4 align:stretch">
-            <td cds-layout="p:lg">
-              <span cds-text="body">${token.name}</span>
-            </td>
-            <td cds-layout="p:lg">${token.formattedValue}</td>
-            <td cds-layout="p:lg">${token.demo}</td>
-          </tr>`;
-        })}
-      </tbody>
-    </table>`;
+    <div cds-layout="display:none display@sm:block">
```

Comment:
I moved this into a wrapper div, the display token was setting the table to display block causing some weirdness with the table not being set as display table.